api/src/test/java/uk/gov/dhsc/htbhf/claimant/message/processor/DetermineEntitlementMessageProcessorTest.java

```diff
@@ -162,6 +176,78 @@ void shouldSuccessfullyProcessMessageAndHandleIneligibleDecision() {
+        LocalDateTime claimStatusTimestamp = messageTimestamp.minus(MAXIMUM_PENDING_EXPIRY_DURATION).minusDays(1);
+        shouldExpireClaimInPendingExpiry(messageTimestamp, claimStatusTimestamp);
+    }
+
+    private void shouldExpireClaimInPendingExpiry(LocalDateTime messageTimestamp, LocalDateTime claimStatusTimestamp) {
+        //Given
+        DetermineEntitlementMessageContext context = buildMessageContextWithClaimInPendingExpiry(claimStatusTimestamp);
+        given(messageContextLoader.loadDetermineEntitlementContext(any())).willReturn(context);
+
+        EligibilityAndEntitlementDecision decision = aDecisionWithStatus(INELIGIBLE);
+        given(eligibilityAndEntitlementService.evaluateClaimantForPaymentCycle(any(), any(), any())).willReturn(decision);
+
+        //Current payment cycle voucher entitlement mocking
+        Message message = aMessageWithTypeAndCreatedTimestamp(DETERMINE_ENTITLEMENT, messageTimestamp);
+
+        //When
+        MessageStatus messageStatus = processor.processMessage(message);
+
+        //Then
+        assertThat(messageStatus).isEqualTo(COMPLETED);
+        assertThat(context.getClaim().getClaimStatus()).isEqualTo(EXPIRED);
+        verify(messageContextLoader).loadDetermineEntitlementContext(message);
+        verify(eligibilityAndEntitlementService).evaluateClaimantForPaymentCycle(context.getClaim().getClaimant(),
+                context.getCurrentPaymentCycle().getCycleStartDate(),
+                context.getPreviousPaymentCycle());
+        verify(claimMessageSender).sendReportClaimMessage(context.getClaim(), decision.getDateOfBirthOfChildren(), UPDATED_FROM_PENDING_EXPIRY_TO_EXPIRED);
+    }
+
+    @Test
+    void shouldNotUpdateClaimInPendingExpiryLessThanMaximumPendingExpiryDuration() {
+        //Given
+        DetermineEntitlementMessageContext context = buildMessageContextWithClaimInPendingExpiry(LocalDateTime.now().minusWeeks(15));
+        given(messageContextLoader.loadDetermineEntitlementContext(any())).willReturn(context);
+
+        EligibilityAndEntitlementDecision decision = aDecisionWithStatus(INELIGIBLE);
+        given(eligibilityAndEntitlementService.evaluateClaimantForPaymentCycle(any(), any(), any())).willReturn(decision);
+
+        //Current payment cycle voucher entitlement mocking
+        Message message = aValidMessageWithType(DETERMINE_ENTITLEMENT);
+
+        //When
+        MessageStatus messageStatus = processor.processMessage(message);
+
+        //Then
+        assertThat(messageStatus).isEqualTo(COMPLETED);
+        assertThat(context.getClaim().getClaimStatus()).isEqualTo(PENDING_EXPIRY);
+        verify(messageContextLoader).loadDetermineEntitlementContext(message);
+        verify(eligibilityAndEntitlementService).evaluateClaimantForPaymentCycle(context.getClaim().getClaimant(),
+                context.getCurrentPaymentCycle().getCycleStartDate(),
+                context.getPreviousPaymentCycle());
+        verifyNoMoreInteractions(claimMessageSender);
```

Comment:
SHouldn't this be `verifyNoInteractions`? Can't see any interaction with this mock.

Reply:
Changed to verifyNoInteractions